### PR TITLE
Add statsd

### PR DESCRIPTION
**Pull request by surminus**
This commit pinches the recipe built here: https://github.com/bernd/fpm-recipes/tree/master/statsd

This version is a couple of years old so may want to look at upgrading it to 0.8? Let me know if you want me to do the work around that.

### CODE REVIEW

File: fpm/recipes/statsd/recipe.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  depends       'nodejs'
+
+  post_install   'post-install'
+  post_uninstall 'post-uninstall'
+
+  config_files '/etc/statsd/config.js'
+
+  def build
+    # Include influxdb backend.
+    safesystem('npm install statsd-influxdb-backend')
```
Comment (deanwilson) [inline fpm/recipes/statsd/recipe.rb]: Do we use this backend? Is it required for package building?

Comment (surminus) [inline fpm/recipes/statsd/recipe.rb]: Will check.

Comment (surminus) [inline fpm/recipes/statsd/recipe.rb]: Concluded we probably don't.

**Comment by surminus** [comment]
This has been updated slightly to use statsd v0.8 (wooo) and also adds a systemd unit file.

**Comment by alexmuller** [comment]
It'd be good to add a maintainer to the fpm recipe, otherwise it uses something (the git user?) on the machine that runs fpm.

**Comment by surminus** [comment]
Added maintainer.

**Pull request merged**